### PR TITLE
tinh

### PR DESCRIPTION
**Pull request by tinhchuquang**
Hello.

### CODE REVIEW

**Comment by chibenwa** [comment]
Hi. Wouldn't you mind solving the conflict?

**Comment by tinhchuquang** [comment]
Yes. Can you show me?

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -11,6 +11,5 @@ Add your name at the bottom of the list:
  - Linh Phan
  - Nguyen Duc Trinh
  - Pham Minh Tuan
- - Vu Ngoc Hoan
- - Bui Thien Bao
+ _ Chu Quang Tinh
```
Comment (chibenwa) [inline CONTRIBUTORS.md]: You removed Vu Ngoc Hoan and Bui Thien Bao. Why?

**Comment by chibenwa** [comment]
See https://github.com/Open-Up/openup2018_01-01/pull/21#issuecomment-377105328

**Pull request merged**